groot-btrfs-release/jobs/groot-btrfs/templates/bin/init-store.erb

```diff
@@ -47,7 +47,24 @@ create_and_mount_filesystem() {
   truncate -s $2 $backing_store_file
 
   mkfs.btrfs -f $backing_store_file
-  mount -t btrfs -o user_subvol_rm_allowed $backing_store_file $1
+
+  # It is possible for the filesystem creation to take some more time
+  # in the background when reaching the mount, causing it to fail.
+  # Simply repeat the attempt. But not too often, i.e. not too long.
+
+  trials=15 ;# 30 seconds, given the sleep period
```

Comment:
How about we add `sleep_time=2` so we don't need this extra comment?

Reply:
Also, we can't be sure it is 30 seconds given the overhead of the command calls.

Reply:
Done with commit b8c6bb018cce208f3a26c646fded7a3250b96554
(I simply say `about ...` now. I.e. I am not bothered by being +/- seconds off in the output)